### ISSUE TITLE
DonPachi - Can't exit service mode

### ISSUE THREAD

**Issue by felleg** [MiSTer-devel/Arcade-Cave_MiSTer#96]
In DonPachi's service mode, "BORD SW ON" and "TEST SW ON" seem to do the exit function, however I don't think these dip switches are exposed in the OSD. 

I tried changing the difficulty -> reset -> service mode, and I notice the setting is reset to default.

**Comment by thehughhefner** [comment]
Yes, so it seems like BORD SW ON and TEST SW ON are supposed to be exit and are not available in the OSD. The current workaround is (when you have done your changes in the service menu), is to select service menu option again in the OSD.

**Comment by nullobject** [comment]
Correct, you just need to hit the "service mode" menu item again.

I could map keyboard keys to the BORD SW ON and TEST SW ON buttons?

**Comment by atrac17** [comment]
Could you do the whole keyboard handler?

**Comment by nullobject** [comment]
> Could you do the whole keyboard handler?

I already have keyboard inputs defined for player controls, is that what you mean? :smile:

**Comment by atrac17** [comment]
Test/Service etc. Last I checked they weren't mapped. That was probably 8 months ago 🤣
